tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql

```diff
@@ -8,17 +8,18 @@ from (
     latestPm.programmeMembershipType,
     latestPm.programmeStartDate,
     latestPm.programmeEndDate,
+    latestPm.curriculumEndDate,
     prg.programmeName,
     prg.owner
   from
     Person p
   join ContactDetails cd on (cd.id = p.id)
   left join GmcDetails gmc on (gmc.id = p.id)
   left join (select distinct pmi.personId, pmi.programmeStartDate, pmi.programmeEndDate,
-          pmi.programmeId, pmi.programmeMembershipType
-          from CurriculumMembership pmi
+          pmi.programmeId, pmi.programmeMembershipType, pmi.curriculumEndDate
+          from ProgrammeMembership pmi
           inner join (select personId, MAX(programmeEndDate) as latestEndDate
-              from CurriculumMembership
+              from ProgrammeMembership
```

Comment:
It seems to me that it is in progress of refactoring programme membership, so @ReubenRobertsHEE helped us to update it to `CurriculumMembership`. 
May I ask what will the refactoring look like in the final stage? If the curriculum fields in the `ProgrammeMembership` table will be deleted, we may need to get programme fields from the `ProgrammeMembership` table and curriculum fields from the `CurriculumMembership` table.

Reply:
https://github.com/Health-Education-England/TIS-TCS/commit/a9b3885d4e64eb7d297dd898e76211bc5e835530
Looks like this was recently intentionally changed for that reason.

Reply:
Ah I confused myself with my outdated test mysql setup, I'll revert, thanks!